cloudpickle/cloudpickle_fast.py

```diff
@@ -337,6 +339,10 @@ def _class_reduce(obj):
     return NotImplemented
 
 
+def _typevar_reduce(obj):
+    return (_make_typevar, _decompose_typevar(obj))
```

Comment:
I think we only want to re-create dynamically created `TypeVar`s. `TypeVar` importable from a module should be pickled as globals. You should be able to use `_is_global` to test the dynamic property of a `TypeVar`.

Reply:
That makes me think that the `_is_global` function is not very well named. A global variable (an element of the `globals()` namespace) is something different.

It think this function should be renamed to something else such as `_is_importable_by_name`.